README.Mac.md

```diff
@@ -1,294 +1,289 @@
-# LORIS Mac Install Guide
-### Mac is no longer supported as of 15.10. 
```

Comment:
Please keep the note saying Mac is no longer supported as a prominent header. 

Also add a note saying that nothing in this document is an official recommendation for LORIS but is only intended to be helpful.

Reply:
@driusan The first few lines of the README illustrate this isn't for production and macOS isn't supported. Do you want me to remove those lines and replace because of redundancy?

Reply:
You can keep them both, one's a header and one's prose.

It doesn't matter if it says it's just for development, it should explicitly say nothing in it is an official recommendation.

Reply:
@driusan okay sounds good.

Reply:
@driusan I added the new change.